crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -135,6 +137,27 @@ def finished?
       current_step == upgrade_steps_6_7.last
     end
 
+    def save_current_node(node_data = {})
```

Comment:
I'm not sure I understand. I think all the node related data fit into the same hash...